### PR TITLE
Installed rpsec

### PR DESCRIPTION
**Pull request by erikwesterberg**
- Setting up unit-testing environment.
- Installed rpsec.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,6 +20,7 @@ group :development do
   gem 'pry-byebug'
   gem 'spring'
   gem 'web-console', '~> 2.0'
+  gem 'rspec-rails'
```
Comment (faraznaeem) [inline Gemfile]: This should go into a test group, please add that

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```
Comment (faraznaeem) [inline spec/rails_helper.rb]: Please clean out the comments

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -0,0 +1,96 @@
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
```
Comment (faraznaeem) [inline spec/spec_helper.rb]: Please clean out the comments

**Comment by erikwesterberg** [comment]
- Added shoulda matchers.
-Cleaned up the files, removed comments.

**Comment by erikwesterberg** [comment]
Cleaned up spec_helper.
alltså added to gemfile: "group :development, :test do"

**Comment by erikwesterberg** [comment]
Changes in gemfile regarding group development test.

**Pull request merged**